src/applications/disability-benefits/all-claims/tests/components/ComboBox.unit.spec.jsx

```diff
@@ -36,111 +40,375 @@ describe('ComboBox', () => {
     };
   });
 
-  describe('Rendering', () => {
-    it('should render listbox with no items by default', () => {
+  describe('Default Rendering', () => {
+    it('should render with label and input', () => {
+      const { getByTestId } = render(<ComboBox {...props} />);
+
+      const input = getByTestId('combobox-input');
+
+      expect(input).to.have.attribute('label', 'Test label');
+      expect(input).to.be.visible;
+    });
+
+    it('should render required', () => {
+      const { getByTestId } = render(<ComboBox {...props} />);
+
+      const input = getByTestId('combobox-input');
+
+      expect(input).to.have.attribute('required');
+    });
+
+    it('should render with no value', () => {
+      const { getByTestId } = render(<ComboBox {...props} />);
+
+      const input = getByTestId('combobox-input');
+
+      expect(input).to.have.value('');
+    });
+
+    it('should render with listbox with no items', () => {
       const { getByRole } = render(<ComboBox {...props} />);
 
       const listbox = getByRole('listbox');
 
       expect(listbox).to.have.length(0);
     });
+  });
 
-    it('should render with initial formData', () => {
+  describe('Updating State', () => {
+    it('should render with value when there is initial formData', () => {
       props.formData = 'initial value';
       const { getByTestId } = render(<ComboBox {...props} />);
 
       const input = getByTestId('combobox-input');
 
-      expect(input.value).to.eq('initial value');
+      expect(input).to.have.value('initial value');
```

Comment:
Just curious, is there any difference between `input.value` `to.eq` and `input` `to.have.value`?

Reply:
No preference here, looks like we use both ways in the codebase, but `to.eq` is much more popular.